service/dap/server.go

```diff
@@ -494,6 +498,8 @@ func (s *Server) onInitializeRequest(request *dap.InitializeRequest) {
 	response.Body.SupportsReadMemoryRequest = false
 	response.Body.SupportsDisassembleRequest = false
 	response.Body.SupportsCancelRequest = false
+	// These requests are not yet supported by vscode-go
+	response.Body.SupportsExceptionInfoRequest = true
```

Comment:
nit: you should move this to the top, grouping together with all the other true capabilities. I don't think the comment is necessary.

Reply:
Done.